specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/examples/renameSubscription.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "subscriptionId": "83aa47df-e3e9-49ff-877b-94304bf3d3ad",
+    "api-version": "2019-03-01-preview",
+    "body": {
+      "SubscriptionName": "Test Sub"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "SubscriptionId": "83aa47df-e3e9-49ff-877b-94304bf3d3ad"
```

Comment:
This doesn't match the schema, it's defined as `value`.

Reply:
Fixed